batch_convert.qml

```diff
@@ -424,22 +428,57 @@ MuseScore {
     // other options
     property alias exportE: exportExcerpts.checked
     property alias travers: traverseSubdirs.checked
+    property alias diffEPath: differentExportPath.checked  // different export path
+    property alias ePath: targetFolderDialog.folder // export path
+    property alias iPath: sourceFolderDialog.folder // import path
     }
 
   FileDialog {
-    id: fileDialog
+    id: sourceFolderDialog
     title: traverseSubdirs.checked?
-      qsTr("Select Startfolder"):
-      qsTr("Select Folder")
+      qsTr("Select sources Startfolder"):
```

Comment:
I'm not an English native, but is there a reason "sources" isn't capitalized as well?

Reply:
Indeed should be title case